js/index.js

```diff
@@ -0,0 +1,36 @@
+$(document).ready(function() {
+
+  $('.readmore,.readless,.learnmore').click(function(event) {
```

Comment:
What you've effectively done is create a click event for all 3 links. Then when you click any of them for the first time, it then does the `event.preventDefault()` and then wires up additional click handlers (lines 11, 12, and 15). The problem here is two fold: (1) The first time you click on any of those links, they won't do anything - at least nothing the user will be able to understand. Is after you click any of them for first time, you now have to click handlers for each link. The first one that calls the `event.preventDefault()` and then the one corresponding handlers on lines 18, 25, or 31. From a user perspective, everything will seem to be working fine. But under the hood, you'll add an additional click handler every single time you click a link.  Its a bit difficult to explain over github. But if you put a `console.log()` before the `event.preventDefault()` and then inside each of the event handlers you'll see how many times those events get called. 

The solution is simple, you don't need the parent click handler on line 3. Then all you need to do is put the `event.preventDefault()` inside each of the 3 click handlers: `showOnClick`, `hideOnClick`, and `learnMoreText`.